Library/Formula/sfml.rb

```diff
@@ -27,9 +17,44 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Patch the bundled libs wording that causes linking errors. Accepted upstream.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  # https://github.com/ludvichzme/SFML/commit/c3f4f62f306f6f9b979b8c0cb2bfb3968712c8fa
+  patch :DATA
 
   def install
-    system "cmake", ".", "-DCMAKE_INSTALL_FRAMEWORK_PREFIX=#{frameworks}/", *std_cmake_args
+    args = std_cmake_args
+    args << "."
+
+    args << "-DSFML_BUILD_DOC=TRUE" if build.with? "doxygen"
+
+    # If we leave this dir present, sfml will always prefer to use its bundled deps
+    # instead of ours, and that breaks absolutely everything.
+    rm_rf buildpath/"extlibs"
+
+    system "cmake", *args
     system "make", "install"
   end
 end
+
+__END__
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b7a99ee..041e6cb 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -267,9 +267,11 @@ if(SFML_OS_WINDOWS)
+ 
+ elseif(SFML_OS_MACOSX)
+ 
+-    # install the non-standard frameworks SFML depends on
++    # Install extlibs dependencies only when used
++    if((${SNDFILE_LIBRARY} EQUAL "${SFML_SOURCE_DIR}/extlibs/libs-osx/Frameworks/sndfile.framework") AND (${FREETYPE_LIBRARY} EQUAL "${SFML_SOURCE_DIR}/extlibs/libs-osx/Frameworks/freetype.framework"))
```

Comment:
I've made a mistake with the operator... it should read STREQUAL instead of EQUAL (twice). 

Proper patch available at https://github.com/LaurentGomila/SFML/commit/992f661ad2fa77dcbcdd5ebfdf3ecb794732b31f (being reviewed)